Add Ctrl + A and Ctrl + Shift + A to (de)select all projects in project manager

Calinou opened the pull request:
These keyboard shortcuts are commonly used in applications to (de)select all entries in a list.

YuriSizov commented on editor/project_manager.cpp:
```diff
@@ -2084,6 +2100,17 @@ void ProjectManager::shortcut_input(const Ref<InputEvent> &p_ev) {
 			} break;
 		}
 
+		if (Input::get_singleton()->is_key_pressed(Key::CMD_OR_CTRL)) {
+			if (k->get_keycode() == Key::A) {
```
Can we use an action instead? (Well, two actions, one for select and one for deselect.) We should have access to editor shortcuts in the PM.

Calinou replied:
I'm not sure if it's really worth adding configurable actions for those standard shortcuts (we already have a few of them here, such as <kbd>Ctrl + F</kbd> just above).

Either way, if we start to do that, we should probably add `ui_select_all` and `ui_deselect_all` (or `ui_select_none`), so that we can use the same shortcut in FileDialog, EditorFileDialog and so on. There's a lot of locations where these shortcuts could be implemented.

Calinou committed:
Add Ctrl + A and Ctrl + Shift + A to (de)select all projects in project manager

These keyboard shortcuts are commonly used in applications to (de)select
all entries in a list.

Calinou commented:
Rebased and tested again, it works as expected.

![image](https://github.com/user-attachments/assets/f6b1cdf9-2678-49d8-b8fc-5ca8acd43ab7)

timothyqiu commented on editor/project_manager.cpp:
```diff
@@ -1018,6 +1018,16 @@ void ProjectManager::shortcut_input(const Ref<InputEvent> &p_ev) {
+					}
+					_update_project_buttons();
+				}
+			}
```
Missing `break` here?

timothyqiu reviewed (approved):
LGTM aside from the issues mentioned.
